DigitalExcellence/dex-frontend
# Implementing design edit project page #98

## Description

Created edit project page and implemented styling. 
Added 'Edit project'-button on project details page.
Added edit project functionality (doesn't fully work yet, must be resolved by the backend)


## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Switch to this branch
2. Go to a project detail page and click on the 'Edit project' button
3. Check if the input fields are filled with the correct project data

## Link to issue

Closes: #92  #98
